README.md

```diff
@@ -9,23 +9,35 @@ Python library to extract text from any file type compatiable with [TIKA](http:/
 - [Apache Tika](http://tika.apache.org/)
 - [Ghostscript](http://www.ghostscript.com/)
 - [Tesseract](https://code.google.com/p/tesseract-ocr/)
+- [Xpdf](http://www.foolabs.com/xpdf/)
 
 ##### Installation
 1. Download tika-app-1.7.jar from [Apache Tika](http://tika.apache.org/)
 2. Mac: `brew install ghostscripts` Ubuntu: `sudo apt-get install ghostscript`
-3. `brew install tesseract` Ubuntu: `sudo apt-get install tesseract-ocr`
+3. Mac: `brew install tesseract` Ubuntu: `sudo apt-get install tesseract-ocr`
+4. Mac: `brew tap homebrew/x11` and `brew install xpdf` Ubuntu: `sudo apt-get install poppler-utils`
 
 ##### Usage
-These script assume that Apache Tika server is running in text mode.
-Start Tika Server
+These script assume that two instances of Tika server are running, one to extract meta data and the other to extract text.
```

Comment:
What? This doesn't make sense. Leave any notions of parallelization out of this library I think.